pulse/broker/src/main/java/gov/ca/emsa/pulse/broker/dao/impl/PatientRecordDAOImpl.java

```diff
@@ -56,7 +57,12 @@ public PatientRecordDTO create(PatientRecordDTO dto) {
 			patient.setDateOfBirth(java.sql.Date.valueOf(dto.getDateOfBirth()));
 		}
 		patient.setSsn(dto.getSsn());
-		patient.setGender(dto.getGender());
+		PatientGenderEntity patientGenderEntity = new PatientGenderEntity();
```

Comment:
Unless this is being done in the controller or manager: 
   Since PatientGender is a fixed set of options, I would just look up the matching PatientGenderEntity rather than expect the ID to be filled in here. You can quickly look it up by code that way the UI only has to send in the code rather than all three (id,code,description) values.